src/resources/postcss/help-hub-iframe.pcss

```
@@ -39,15 +39,52 @@ body {
 	transition: var(--tec-transition-opacity);
 }
 
+/**
+Zendesk Styles
+ */
 .zendesk-widget-open * {
 	pointer-events: none;
 }
 
-iframe[name="Messaging window"] {
-	margin-right: -16px;
-	z-index: 99999999 !important; /* Make sure the Zendesk Iframe is above DocBot */
+/* Overwrite inline Iframe styles from Zendesk */
+#webWidget {
+	height: 100% !important;
+	max-width: 95% !important;
+	min-width: 33vw !important;
+	z-index: 99999999 !important;
+	margin: 0 auto !important;
+	width: 50% !important; /* Default width */
 }
 
+/* Above 700px: 50% width, centered */
+@media (min-width: 700px) {
+	#webWidget {
+		width: 50% !important;
+		margin: 0 auto !important;
+	}
+}
+
+/* Between 350px and 700px: 80% width with 10% margins */
+@media (max-width: 700px) and (min-width: 351px) {
```

These numbers are weird, but:
```suggestion
@media (min-width: 351px) and (max-width: 700px) {
```
min before max please

---

Also - who has a screen smaller than 350px? Is that necessary? (our smallest breakpoint is 320px though...)